lib/constants.js

```diff
@@ -69,3 +78,9 @@ exports.THING_TYPE_ON_OFF_LIGHT = 'onOffLight';
 exports.THING_TYPE_DIMMABLE_LIGHT = 'dimmableLight';
 exports.THING_TYPE_ON_OFF_COLOR_LIGHT = 'onOffColorLight';
 exports.THING_TYPE_DIMMABLE_COLOR_LIGHT = 'dimmableColorLight';
+
+exports.NotificationLevel = {
+  LOW: 0,
+  NORMAL: 1,
+  HIGH: 2,
```

Comment:
I should know this already, but is there a similar set of urgency levels for alarm events we could mirror here?

Reply:
No, we never decided on levels for the alarm property/event, so it doesn't exist in the schema.